build/lifecycle.go

```diff
@@ -50,16 +57,34 @@ type LifecycleOptions struct {
 }
 
 func (l *Lifecycle) Execute(ctx context.Context, opts LifecycleOptions) error {
-	cacheImage := cache.New(opts.Image, l.docker)
-	l.logger.Verbose("Using cache image %s", style.Symbol(cacheImage.Image()))
+	l.Setup(opts)
+	defer l.Cleanup()
+
+	lifecycle020OrLater := l.hasVersion020OrLater()
```

Comment:
maybe we can call this check `supportsLaunchCache` or something like that instead of `hasVersion0200OrLater`

Reply:
Will do!